Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM debian:stretch
+
+RUN apt-get -qq update \
+    && apt-get -qq install -y --no-install-recommends \
+       build-essential \
+       wget \
+       clang \
+       gcc \
+       ca-certificates \
+       && rm -rf /var/lib/apt/lists/* \
+       && wget -q https://cmake.org/files/v3.11/cmake-3.11.4-Linux-x86_64.tar.gz \
```

Comment:
The built-in CMake version on stretch should be high enough for this project.  So just add `cmake` to the apt-get list. No need to install a special download.